app/helpers/ugc_form_helper.rb

```diff
@@ -40,7 +40,7 @@ def aasm_events_for_select(events)
   def edm_rights_options
     %w(http://creativecommons.org/publicdomain/mark/1.0/ http://creativecommons.org/licenses/by-sa/4.0/ http://rightsstatements.org/vocab/CNE/1.0/).map do |url|
         license = cc_license_from_url(url)
-        [edm_rights_label_html(cc_license_i18n_key(license)), license.id]
+        [edm_rights_label_html(cc_license_i18n_key(license)), license.id, 'data-license-url': url]
```

Comment:
What does the `data-license-url` attribute do? I see no difference to display with/without it.

Reply:
it's added so that the license icons can be derived from it and the license can be linked to.
It should appear on each radio option.

Reply:
License data stored in [javascript](https://github.com/europeana/Europeana-Patternlab/blob/develop/source/js/modules/eu/channels/ugc.js#L408-L421) gets written to the dom.

Reply:
Also: if I revert the file "config/initializers/simple_form.rb" locally I get some of the display issues you've been having.